### ISSUE TITLE
Show pfm_title rather than pfm_name when using pfm_exclude

### ISSUE THREAD

**Issue by apizz** [ProfileCreator/ProfileCreator#106]
When using `pfm_exclude`, if a preference requires the existence of another preference, or that another preference have a particular value, what's displayed in the exclusion is the `pfm_name`.

In cases where you're using `pfm_title`, it makes it a bit difficult to find the preference, as you have to click the displayed preference name to reveal the preference key's `pfm_name`.  Examples below.
  
![screen shot 2018-12-31 at 12 16 27 am](https://user-images.githubusercontent.com/11789931/50554943-43935b80-0c92-11e9-9feb-052ba6b43170.png)
![screen shot 2018-12-31 at 12 16 40 am](https://user-images.githubusercontent.com/11789931/50554945-48580f80-0c92-11e9-9251-f32459bce4e5.png)

It would be helpful then where a preference has a `pfm_title` to have this shown in the exclusion, rather than the `pfm_name`, so it's easier to find.

While not necessary for this issue, it might be nice to have "bread crumbs" that can more easily direct folks to the location of the related preference(s) when an exclusion rule is met.

**Comment by apizz** [comment]
This is also important when a preference from one domain requires a preference from another (`pfm_domain`) in order to determine which additional payload and preference is required to satisfy the rule.

**Comment by erikberglund** [comment]
Yes, I agree. And also, these damn Optional() swift artifacts are so annoying, I have made them go away in several places but not all yet.

**Comment by apizz** [comment]
Is this addressed in upcoming release?
